english/blah-balh/blah-blah.md

```diff
@@ -4,4 +4,6 @@ Be as smart as you can be in your voice so you don't sound stupid.
 
 And sing if possible. Singing cheers people up.
 
-If no sing [go back](../marshmallow.md)
\ No newline at end of file
+If no sing [go back](../marshmallow.md)
+
+Or go [drink a coffee](https://github.com/udacity/create-your-own-adventure/tree/master/english/coffee) to get your courage up
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.